core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/store/dot-container-list.store.ts

```diff
@@ -78,41 +77,33 @@ export class DotContainerListStore extends ComponentStore<DotContainerListState>
 
                     this.paginatorService.setExtraParams('host', identifier);
 
-                    return forkJoin([
-                        this.route.data.pipe(pluck('dotContainerListResolverData'), take(1)),
-                        this.paginatorService.get()
-                    ]);
+                    return this.route.data.pipe(pluck('dotContainerListResolverData'), take(1));
                 })
             )
-            .subscribe(
-                ([[isEnterprise, hasEnvironments], containers]: [
-                    [boolean, boolean],
-                    DotContainer[]
-                ]) => {
-                    this.setState({
-                        containerBulkActions: this.getContainerBulkActions(
-                            hasEnvironments,
-                            isEnterprise
-                        ),
-                        tableColumns: this.getContainerColumns(),
-                        stateLabels: this.getStateLabels(),
-                        isEnterprise: isEnterprise,
-                        hasEnvironments: hasEnvironments,
-                        addToBundleIdentifier: '',
-                        selectedContainers: [],
-                        actionHeaderOptions: this.getActionHeaderOptions(),
-                        listing: {} as DotListingDataTableComponent,
-                        notifyMessages: {
-                            payload: {},
-                            message: null,
-                            failsInfo: []
-                        } as DotNotifyMessages,
-                        containers,
-                        maxPageLinks: this.paginatorService.maxLinksPage,
-                        totalRecords: this.paginatorService.totalRecords
-                    });
-                }
-            );
+            .subscribe(([isEnterprise, hasEnvironments]: [boolean, boolean]) => {
+                this.setState({
+                    containerBulkActions: this.getContainerBulkActions(
+                        hasEnvironments,
+                        isEnterprise
+                    ),
+                    tableColumns: this.getContainerColumns(),
+                    stateLabels: this.getStateLabels(),
+                    isEnterprise: isEnterprise,
+                    hasEnvironments: hasEnvironments,
+                    addToBundleIdentifier: '',
+                    selectedContainers: [],
+                    actionHeaderOptions: this.getActionHeaderOptions(),
+                    listing: {} as DotListingDataTableComponent,
+                    notifyMessages: {
+                        payload: {},
+                        message: null,
+                        failsInfo: []
+                    } as DotNotifyMessages,
+                    containers: [],
+                    maxPageLinks: 5,
```

Comment:
Can we create a const for this, looks like something we can change in the future, also check if the test are working